src/ports/postgres/modules/deep_learning/madlib_keras_helper.py_in

```diff
@@ -0,0 +1,96 @@
+# coding=utf-8
```

Comment:
IMO, we shouldn't name this file `madlib_keras_helper.py_in` because this will just become a catch-all for all things related to deep learning. Naming it helper makes it harder to navigate through code and maintain it in the long term. For ex the function `expand_input_dims` can be in a file called something along the lines`all_things_numpy.py_in` and this file can be called `predict_input_params.py_in`.

Having said all this, I will leave the final decision up to you to decide and even if you plan to change this we can do it in a future PR.

Reply:
Agree and disagree. `predict_input_params.py_in` makes sense, and I have made the necessary changes for that. But for the other function, creating a new file called `all_things_numpy` may not be a good idea since we do have a lot of functions that use numpy in a different file. So that filename will be a misnomer. If we decide to name it something else, then there will be just that one function in that file, and we will also have to create a new file for the colnames variables. I'd rather have them both together in `madlib_keras_helper.py_in`. IMO, If we add more functions in future PRs, we should create appropriate files if it makes sense, else it's ok to add it in the helper file if we cannot classify it cleanly to a separate file.